src/file_upload.rs

```diff
@@ -111,23 +118,17 @@ pub fn upload_file(req: &HttpRequest<crate::MiniserveConfig>) -> FutureResponse<
             }
         }
         Err(_) => {
-            return Box::new(future::ok(
-                HttpResponse::BadRequest().body("Unspecified parameter path"),
```

Comment:
I think we should keep this as a `BadRequest` as it seems to be the most semantically correct error here.

Reply:
Yes, it seems to be the best.